number of configurations
36
  0%|          | 0/36 [00:00<?, ?it/s]  0%|          | 0/36 [00:00<?, ?it/s]
Traceback (most recent call last):
  File "main.py", line 54, in <module>
    results = run(imp, results, args.job_name, args.gpu)
  File "/Users/shreyasshandilya/Desktop/Shandilya/Padhai/CS6886/assignment7/run.py", line 72, in run
    test.build(config)
  File "/Users/shreyasshandilya/Desktop/Shandilya/Padhai/CS6886/assignment7/test.py", line 18, in build
    self.layer = self.layer.cuda()
  File "/Users/shreyasshandilya/Desktop/Shandilya/Padhai/CS6886/assignment7/venv/lib/python3.7/site-packages/torch/nn/modules/module.py", line 463, in cuda
    return self._apply(lambda t: t.cuda(device))
  File "/Users/shreyasshandilya/Desktop/Shandilya/Padhai/CS6886/assignment7/venv/lib/python3.7/site-packages/torch/nn/modules/module.py", line 381, in _apply
    param_applied = fn(param)
  File "/Users/shreyasshandilya/Desktop/Shandilya/Padhai/CS6886/assignment7/venv/lib/python3.7/site-packages/torch/nn/modules/module.py", line 463, in <lambda>
    return self._apply(lambda t: t.cuda(device))
  File "/Users/shreyasshandilya/Desktop/Shandilya/Padhai/CS6886/assignment7/venv/lib/python3.7/site-packages/torch/cuda/__init__.py", line 166, in _lazy_init
    raise AssertionError("Torch not compiled with CUDA enabled")
AssertionError: Torch not compiled with CUDA enabled
